Multisig/UI/Transactions/TransactionsList/TransactionsViewModel.swift

```diff
@@ -101,4 +109,11 @@ class TransactionsViewModel: BasicLoadableViewModel {
     func isLast(transaction: TransactionViewModel) -> Bool {
         transactionsList.lastTransaction == transaction
     }
+
+    func creationTransaction() throws -> CreationTransactionViewModel? {
+        if self.nextURL == nil {
```

Comment:
this should be  changed to `guard ... else { return nil }`  to  make it more Swifty